.travis.yml

```diff
@@ -19,4 +19,10 @@ matrix:
     - rvm: 2.1.9
       gemfile: gemfiles/rails3222.gemfile
 
-before_install: gem install bundler -v 1.13.6
+before_install:
+  - gem install bundler -v 1.13.6
+  - nvm install node
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
Is it a convenient way to install yarn for Travis? I've found an official advice. https://github.com/travis-ci/travis-ci/issues/6720#issuecomment-254988120

Actually, I'm not sure about the best way here